src/runtime/offers-flow.js

```diff
@@ -120,6 +126,9 @@ export class SubscribeOptionFlow {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    /** @private @const {boolean} */
+    this.isOffersViewClosable_ = false;
```

Comment:
Hmm. Not seeing it set anywhere.

Reply:
Used in line #179 as a default value if not available in options.

Reply:
Hmm. I still don't understand why the private var is needed? Maybe just define a top-level `const DEFAULT_CLOSABLE = false`?

Reply:
Kept private var here so that individual views (AbbrvView, SubscribeOptionsView) can have different values in future (though currently both are same).

Moved to `const` for now.